cmsl1t/plotting/resolution.py

```diff
@@ -81,37 +90,46 @@ def overlay_with_emu(self, emu_plotter, with_fits=False):
         for (pile_up, ), hist in self.plots.flat_items_all():
             if pile_up == bn.Base.everything:
                 hist.SetLineStyle(1)
-                hist.drawstyle = "hist"
+                hist.drawstyle = ResolutionPlot.drawstyle
                 label = "HW, all PU"
             else:
                 continue
-            hist.SetLineWidth(2)
+            hist.SetMarkerSize(0.5)
+            hist.SetLineWidth(1)
             hists.append(hist)
             labels.append(label)
 
         for (pile_up, ), hist in emu_plotter.plots.flat_items_all():
             if pile_up == bn.Base.everything:
                 hist.SetLineStyle(1)
-                hist.drawstyle = "hist"
+                hist.drawstyle = ResolutionPlot.drawstyle
                 label = "Emu, all PU"
             else:
                 continue
-            hist.SetLineWidth(2)
+            hist.SetMarkerSize(0.5)
+            hist.SetLineWidth(1)
             hists.append(hist)
             labels.append(label)
 
         self.__make_overlay(hists, fits, labels,
                             "Number of events", "__Overlay_Emu")
 
-        integral = hist.integral()
-        normed_hists = [hist / integral if integral !=
-                        0 else hist.Clone() for hist in hists]
+        normed_hists = [hist.Clone() for hist in hists]
+        for hist in normed_hists:
+            if hist.integral != 0:
+                hist = hist / hist.integral()
```

Comment:
Same comment as above: is `hist.integral` a function or just a value (possibly returned as an `@property`)?  Even if both are valid code, its nicer to stick to just one.